core/primitives/ss58_codec.cpp

```diff
@@ -69,10 +69,16 @@ namespace kagome::primitives {
                          const crypto::Hasher &hasher) noexcept {
     common::Buffer ss58_bytes;
     ss58_bytes.reserve(kSs58Length);
-    ss58_bytes.putUint8(account_type).put(id);
+    if(account_type < 64) {
+      ss58_bytes.putUint8(account_type).put(id);
+    }else{
+// https://docs.substrate.io/fundamentals/accounts-addresses-keys/
```

Comment:
```suggestion
    if (account_type < 64) {
      ss58_bytes.putUint8(account_type).put(id);
    } else {
      // https://docs.substrate.io/fundamentals/accounts-addresses-keys/
```
in accordance format convention